.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Extension Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Build and compress
+        run: |
+          npm install
+          npm run build
+          zip -r extension-chrome.zip dist
+          rm -rf dist
+          npm run build_ff
+          zip -r extension-firefox.zip dist
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
```

Comment:
Let's hardcode this to use version `@v1.2.1`, we want to make this workflow deterministic when we can.